pom.xml

```diff
@@ -1544,16 +1458,6 @@
         <version>${bouncycastle.version}</version>
         <scope>test</scope>
       </dependency>
-      <!-- Managed up to match Hadoop in HADOOP-16530 -->
-      <!--
-        When upgrading `xercesImpl` version, also need to change
-        the version definition in `SparkBuild#DependencyOverrides`.
-      -->
-      <dependency>
-        <groupId>xerces</groupId>
-        <artifactId>xercesImpl</artifactId>
```

Comment:
Is it because Apache Spark doesn't require the `xerces:xercesImpl` dependency?

Reply:
If that's the case, should we clean up the corresponding binary LICENSE of `xerces:xercesImpl` ?

Reply:
it's a transitive dep of hadoop vanilla client, spark does not directly use it.

Reply:
right, updated LICENSE and NOTICE

Reply:
It would be best to declare this change in the pr description.

Reply:
updated